sdk/communication/azure-communication-identity/tests/test_identifier_raw_id.py

```diff
@@ -8,6 +8,8 @@
 import pytest
 import unittest
 from azure.communication.identity import *
+from azure.communication.identity._shared.models import MicrosoftBotIdentifier
```

Comment:
We shouldn't need to import from a private namespace to test.

Reply:
Removed this.